<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登入系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">
            QLabel#label {
                font: 22pt &quot;幼圆&quot;;
                font-weight: bold;
                qproperty-alignment: AlignCenter;
            }
        </string>
       </property>
       <property name="text">
        <string>快递物流管理系统</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="phoneInput">
       <property name="styleSheet">
        <string notr="true">
            QLineEdit {
                padding: 8px;
                font-size: 14px;
                border: 1px solid #ccc;
                border-radius: 4px;
            }
    </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="phtonLabel">
       <property name="styleSheet">
        <string notr="true">
            QLabel {
                font: 12pt &quot;幼圆&quot;;
                font-weight: bold;
            }
        </string>
       </property>
       <property name="text">
        <string>手机号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pwdInput">
       <property name="styleSheet">
        <string notr="true">
            QLineEdit {
                padding: 8px;
                font-size: 14px;
                border: 1px solid #ccc;
                border-radius: 4px;
            }
        </string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pwdLabel">
       <property name="styleSheet">
        <string notr="true">
            QLabel {
                font: 12pt &quot;幼圆&quot;;
                font-weight: bold;
            }
        </string>
       </property>
       <property name="text">
        <string>密码:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="confirmPwdLabel">
       <property name="styleSheet">
        <string notr="true">
            QLabel {
                font: 12pt &quot;幼圆&quot;;
                font-weight: bold;
            }
        </string>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="confirmPwdInput">
       <property name="styleSheet">
        <string notr="true">
            QLineEdit {
                padding: 8px;
                font-size: 14px;
                border: 1px solid #ccc;
                border-radius: 4px;
            }
    </string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="registerBtn">
     <property name="styleSheet">
      <string notr="true">
        QPushButton {
            padding: 8px 16px;
            font: 15px &quot;幼圆&quot;;
            font-weight: bold;
            border: 1px solid #4CAF50;
            border-radius: 4px;
            color: #fff;
            background-color: #4CAF50;
        }

        /* 鼠标悬停时按钮样式 */
        QPushButton:hover {
            background-color: #45a049;
            border: 1px solid #45a049;
        }
        </string>
     </property>
     <property name="text">
      <string>确认注册</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
